### PR TITLE
Addon signing min length fix

### PR DESCRIPTION
**Pull request by rtilder**

### CODE REVIEW

File: trunion/validators.py [hunk 1]
```diff
@@ -164,12 +164,14 @@ def valid_addon(request):
...
+    if len(request.POST['addon_id']) < 1:
+        raise HTTPBadRequest('addon_id is very short(<1 byte): "%s"'
                              % request.POST['addon_id'])
 
-    if len(request.POST['addon_id']) > 128:
-        raise HTTPBadRequest('addon_id is very long(>128 bytes): "%s"'
+    # 64 byte length limit is a function of the X509 spec's definition of the
+    # CNAME attribute.
+    if len(request.POST['addon_id']) > 64:
+        raise HTTPBadRequest('addon_id is very long(>64 bytes): "%s"'
```
Comment (andymckay) [inline trunion/validators.py]: remind me, its olympia that will hash? it if its longer than 64?

**Comment by andymckay** [comment]
r+ wc

File: trunion/validators.py (hunk 1 above)
Comment (rtilder) [inline trunion/validators.py]: Correct and Firefox has the logic to check for the hash if the ID is > 64.

**Comment by bhouston** [comment]
Why am I getting email notifications for this bug of this project?  I didn't watch this nor am I related to this project in any way.  Maybe a github error.  Strange.

**Comment by bhouston** [comment]
Ah, I see.  I'll fix it, sorry for the noise.

**Pull request merged**